src/mlpack/tests/akfn_test.cpp

```diff
@@ -163,6 +163,12 @@ TEST_CASE("AKFNSingleCoverTreeTest", "[AKFNTest]")
  *
  * Errors are produced if the results are not according to relative error.
  */
+
+// These macros are added in order to stop executing these tests on aarch64.
+// These is unknown issue with Dual Cover tree test on these architecture
```

Comment:
Looks like you meant to write it's a known issue, also there is a stop missing at the end.

Reply:
:+1: